pkg/streaming/commands/streaming.go

```diff
@@ -29,7 +29,10 @@ func NewStreamingCommand(ctx context.Context, c *cli.Config) *cobra.Command {
 		Use:   "streaming",
 		Short: "(experimental) streaming runtime for riff functions",
 		Long: strings.TrimSpace(`
-<todo>
+The streaming runtime uses riff functions, processor and stream custom resources
```

Comment:
"riff" should never be used in command text.

```suggestion
The streaming runtime uses ` + c.Name + ` functions, processor and stream custom resources
```